tools/depcheck/pkg/cmd/trace/filter_test.go

```diff
@@ -0,0 +1,328 @@
+package trace
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/gonum/graph/concrete"
+
+	depgraph "github.com/openshift/origin/tools/depcheck/pkg/graph"
+)
+
+type testNode struct {
+	name     string
+	children []string
+}
+
+func getVendorNodes() []*testNode {
+	return []*testNode{
+		{
+			name: "github.com/test/repo/vendor/github.com/testvendor/vendor_one",
+			children: []string{
```

Comment:
children?  perhaps `outboundEdges`

You know, you don't get paid extra for long names.....